src/app/components/account/account-avatar/account-avatar.tsx

```diff
@@ -4,25 +4,9 @@ import { Box, CircleProps } from 'leather-styles/jsx';
 
 import { DynamicColorCircle } from '@app/ui/components/dynamic-color-circle';
 
-const getAvatarText = (name: string, index: number) => {
-  // Returns a string with the account's ordinal number.
-  //
-  // Assumes that account names starting with "Account" have been autogenerated
-  // by the app. This will probably break when the account has a BNS name
-  // starting with "Account", although it's probably good enough for now.
-  if (name.startsWith('Account')) return String(index + 1);
-
-  // At the time of writing, Firefox does not yet support Segmenter. Avatar
-  // text will default to the account's ordinal number.
-  if (!Intl.Segmenter) {
-    return String(index + 1);
-  }
-
-  // Displays the first grapheme of the account name in uppercase.
-  //
-  // Using graphemes allows proper handling of emojis and other characters
-  // with multiple code points.
-  return [...new Intl.Segmenter().segment(name)][0].segment.toUpperCase();
+const getAvatarText = (index: number) => {
+  // Always return account number in the Account Circle
```

Comment:
I think instead of leaving this comment, you might want to just rename the function, `getAccountNumber()`?